integration-intygsadmin/src/main/java/se/inera/intyg/minaintyg/integration/intygsadmin/BannerConverter.java

```diff
@@ -0,0 +1,29 @@
+package se.inera.intyg.minaintyg.integration.intygsadmin;
+
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.stereotype.Component;
+import se.inera.intyg.minaintyg.integration.api.banner.model.Banner;
+import se.inera.intyg.minaintyg.integration.intygsadmin.client.dto.BannerDTO;
+
+@Component
+public class BannerConverter {
+
+  public List<Banner> convert(BannerDTO[] banners) {
+    return Arrays.stream(banners)
+        .map(BannerConverter::toBanner)
+        .toList();
+  }
+
+  private static Banner toBanner(BannerDTO bannerDTO) {
+    return Banner.builder()
```

Comment:
Do we want to send all of these values to the application layer? I was thinking that some of these values would be used to filter out which banners we actually want to send.